client/my-sites/plans-features-main/components/plan-notice-credit-update.tsx

```diff
@@ -16,13 +17,7 @@ type Props = {
 	visiblePlans?: PlanSlug[];
 };
 
-const PlanNoticeCreditUpgrade = ( {
-	className,
-	onDismissClick,
-	linkTarget,
```

Comment:
I think we need to remove this prop in all callers.

Reply:
Make sense. I'll clean it up.